src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,110 @@
+package core.basesyntax;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+
+public class RegistrationServiceTest {
+    private static RegistrationService registrationService;
+    private static User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        Storage.people.clear();
+        user.setLogin("userName");
+        user.setPassword("adaswwgwef");
+        user.setAge(33);
+    }
+
+    @Test
+    void register_validUser_Ok() {
+        User actual = registrationService.register(user);
+        User expected = user;
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    void register_nullUser_NotOk() {
+        user = null;
+        assertThrows(RuntimeException.class, () -> registrationService.register(user));
```

Comment:
```suggestion
        assertThrows(RuntimeException.class, () -> registrationService.register(null));
```
чтобы тесты не завалились после переноса инициализации юзера в beforeAll